src/me/truemb/rentit/utils/UtilMethodes.java

```diff
@@ -43,9 +44,11 @@
 public class UtilMethodes {
 
 	private Main instance;
+	private ChestsUtils chestsUtils;
 
 	public UtilMethodes(Main plugin) {
 		this.instance = plugin;
+		this.chestsUtils = new ChestsUtils(plugin);
```

Comment:
Everything was created inside it's own `ChestUtils` but I kept the API unchanged. We could update the consumer `ShopListener` to use chest utils directly if we add it to the main plugin instead of inside here.

Reply:
It is a good idea. I am planing to remove this UtilMethodes Class in the future anyway, since it got way to big. More classes with the exact purpose and smaller size are great!

Reply:
Makes sense. I have exposed the ChestUtils directly in the plugin and removed those methods from the original place in UtilMethodes. Both `ShopListener` and `ShopCacheFileManager` now access it.